boards/common/weact-f4x1cx/Makefile.dep

```diff
@@ -13,3 +13,9 @@ endif
 ifneq (,$(filter mtd,$(USEMODULE)))
   USEMODULE += mtd_spi_nor
 endif
+
+# default to using littlefs2 on the external flash
+ifneq (,$(filter vfs_default,$(USEMODULE)))
```

Comment:
Just to be sure my understanding is up to date: `vfs_default` is not intended to become a default module, is it? Since the external flash is not actually present on the board (at least in the configuration I bought), but only a footprint to solder it to (which I have done for mine :)).

We should avoid that this board tries to mount the external flash without the application / user indicating in some way that flash was soldered in. But if I'm up to speed, `vfs_default` is not intended to become a default module and explicitly adding it can be considered as the user confirming to have provided the flash, right?

(The same would also apply for the boards with an SD card reader, since you would have to insert a card prior to be able to use it.)

Reply:
Yea `vfs_default` is not a default module, it is more like `netdev_default` in that it selects all that is needed to get a file system on the selected board (if the board provides it).